client/js/components/CreateGameDialog.vue

```diff
@@ -17,7 +17,14 @@
         <h2>Create Game</h2>
       </v-card-title>
       <v-card-text>
-        <v-switch v-model="isRanked" label="Ranked" data-cy="create-game-ranked-switch" />
+        <div class="d-flex align-center">
+          <div>
+            <v-switch v-model="isRanked" label="Ranked" data-cy="create-game-ranked-switch" />
+          </div>
+          <div>
+            <stats-scoring-dialog :show-text="false" data-cy="ranked-info-button" />
+          </div>
```

Comment:
Any reason for the wrapping `div`s?
```suggestion
          <v-switch v-model="isRanked" label="Ranked" data-cy="create-game-ranked-switch" />
          <stats-scoring-dialog :show-text="false" data-cy="ranked-info-button" />
```

Reply:
huh, thought I tried that and they ended up getting stacked vertically.  Guess not!